src/app/view/components/_common/dot-page-selector/dot-page-selector.component.ts

```diff
@@ -57,9 +62,31 @@ export class DotPageSelectorComponent implements ControlValueAccessor {
      * @param DotPageAsset item
      * @memberof DotPageSelectorComponent
      */
-    onSelect(item: DotPageAsset): void {
-        this.selected.emit(item);
-        this.propagateChange(item.identifier);
+    onSelect(item: DotPageSeletorItem): void {
+        if (this.results.type === 'site') {
+            const site: Site = <Site>item.payload;
+            this.dotPageSelectorService.setCurrentHost(site);
+            this.resetResults();
```

Comment:
don't we need do a propagateChange here?, maybe something like: this.propagateChange(null);

I don't know if this one is the same:
https://github.com/dotCMS/core-web/pull/856/files#diff-0913f18fc1fe29a460899c0712b774b5R111